src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -237,6 +245,9 @@ def delete_acl_policy(client, container_name, policy_name, **kwargs):
     ''' Delete a stored access policy on a containing object '''
     acl = _get_acl(client, container_name, **kwargs)
     del acl[policy_name]
+    if hasattr(acl, 'public_access'):
```

Comment:
I would argue we should open a bug against the SDK (or re-open the existing bug that they closed) for this and annotate this as what it is: a workaround. Especially given that the SDK is hand authored, there's no reason to burden every client application with this service limitation flaw.